x-pack/plugins/ml/public/application/timeseriesexplorer/timeseriesexplorer_help_popover.tsx

```diff
@@ -27,7 +27,7 @@ export const TimeSeriesExplorerHelpPopover = () => {
       <p>
         <FormattedMessage
           id="xpack.ml.timeSeriesExplorer.popoverAnomalyExplanation"
-          defaultMessage="An anomaly score is calculated for each bucket time interval, with a value from 0 to 100. Anomalous events are highlighted in colors that indicate their severity. If an anomaly is depicted with a cross symbol instead of a dot, it has a medium or high multi-bucket impact. This extra analysis can catch anomalies even when they fall within the bounds of expected behavior."
+          defaultMessage="An anomaly score is calculated for each bucket time interval, with a value from 0 to 100. Anomalous events are highlighted in colors that indicate their severity. If an anomaly is depicted with a cross symbol instead of a dot, it has a moderate, significant or high multi-bucket impact. This extra analysis can catch anomalies even when they fall within the bounds of expected behavior."
```

Comment:
Not sure if there should be another comma after 'significant' or not for clarity 🤔 To signify there are three levels of impact.

Reply:
Added the Oxford comma after consultation with @szabosteve in https://github.com/elastic/kibana/pull/146866/commits/aecb00b038199a7bd88cb7da27e486a2c06ff78b